apps/dashboard/src/@/components/ui/form.tsx

```diff
@@ -112,12 +112,18 @@ const RequiredFormLabel = React.forwardRef<
     <ToolTipLabel label="Required">
       <Label
         ref={ref}
-        className={cn(error && "text-destructive", "relative", className)}
+        className={cn(
+          error && "text-destructive-foreground",
```

Comment:
We should use `text-destructive-text` here instead ( I have a PR open that does this change - https://github.com/thirdweb-dev/js/pull/3829 )